src/pages/messages/components/NviStatusFilter.tsx

```diff
@@ -60,6 +62,7 @@ export const NviStatusFilter = () => {
       <MenuItem value={'approved' satisfies NviCandidateStatus}>{t('tasks.nvi.status.Approved')}</MenuItem>
       <MenuItem value={'rejected' satisfies NviCandidateStatus}>{t('tasks.nvi.status.Rejected')}</MenuItem>
       <MenuItem value={'dispute' satisfies NviCandidateGlobalStatus}>{t('tasks.nvi.status.Dispute')}</MenuItem>
+      <MenuItem value={'all'}>{t('tasks.nvi.status.all')}</MenuItem>
```

Comment:
Vi har allerede en oversettelse for dette som vi kan gjenbruke i stedet for å legge inn en ny som er lik
```suggestion
      <MenuItem value={'all'}>{t('common.show_all')}</MenuItem>
```

Reply:
Tenker det gir mening å følge opp denne kommentaren, istede for å legge inn en duplikat nøkkel.